torch/csrc/utils/tensor_new.h

```diff
@@ -29,105 +29,105 @@ namespace torch::utils {
 TORCH_API bool only_lift_cpu_tensors();
 TORCH_API void set_only_lift_cpu_tensors(bool value);
 
-at::Tensor base_tensor_ctor(PyObject* args, PyObject* kwargs);
-at::Tensor legacy_tensor_ctor(
+TORCH_PYTHON_API at::Tensor base_tensor_ctor(PyObject* args, PyObject* kwargs);
+TORCH_PYTHON_API at::Tensor legacy_tensor_ctor(
     c10::DispatchKey dispatch_key,
     at::ScalarType scalar_type,
     PyObject* args,
     PyObject* kwargs);
-at::Tensor legacy_tensor_new(
+TORCH_PYTHON_API at::Tensor legacy_tensor_new(
     c10::DispatchKey dispatch_key,
     at::ScalarType scalar_type,
     PyObject* args,
     PyObject* kwargs);
-at::Tensor indexing_tensor_from_data(
+TORCH_PYTHON_API at::Tensor indexing_tensor_from_data(
     c10::TensorOptions options,
     at::ScalarType scalar_type,
     std::optional<at::Device> device,
     PyObject* data);
-at::Tensor sparse_coo_tensor_ctor(
+TORCH_PYTHON_API at::Tensor sparse_coo_tensor_ctor(
     c10::DispatchKey dispatch_key,
     at::ScalarType scalar_type,
     PythonArgs& r);
-void _validate_sparse_coo_tensor_args(
+TORCH_PYTHON_API void _validate_sparse_coo_tensor_args(
     c10::DispatchKey dispatch_key,
     at::ScalarType scalar_type,
     PyObject* args,
     PyObject* kwargs);
 
-at::Tensor sparse_compressed_tensor_ctor(
+TORCH_PYTHON_API at::Tensor sparse_compressed_tensor_ctor(
     c10::DispatchKey dispatch_key,
     at::ScalarType scalar_type,
     PythonArgs& r);
-at::Tensor sparse_csr_tensor_ctor(
+TORCH_PYTHON_API at::Tensor sparse_csr_tensor_ctor(
     c10::DispatchKey dispatch_key,
     at::ScalarType scalar_type,
     PythonArgs& r);
-at::Tensor sparse_csc_tensor_ctor(
+TORCH_PYTHON_API at::Tensor sparse_csc_tensor_ctor(
     c10::DispatchKey dispatch_key,
     at::ScalarType scalar_type,
     PythonArgs& r);
-at::Tensor sparse_bsr_tensor_ctor(
+TORCH_PYTHON_API at::Tensor sparse_bsr_tensor_ctor(
     c10::DispatchKey dispatch_key,
     at::ScalarType scalar_type,
     PythonArgs& r);
-at::Tensor sparse_bsc_tensor_ctor(
+TORCH_PYTHON_API at::Tensor sparse_bsc_tensor_ctor(
     c10::DispatchKey dispatch_key,
     at::ScalarType scalar_type,
     PythonArgs& r);
 
-void _validate_sparse_compressed_tensor_args(
+TORCH_PYTHON_API void _validate_sparse_compressed_tensor_args(
```

Comment:
The _ prefix marks them as internal functions. If they are really required, should they be renamed?